docs/xdr/features/investigate/query_builder.md

```diff
@@ -71,21 +71,41 @@ When using the Query Builder with Alerts as a source, users can filter and manip
 | Creation date             | The date and time when the alert was initially created.                                     |
 | Update date               | The date and time when the alert was last updated.                                          |
 | Entity UUID               | A unique identifier representing the entity associated with the alert.                     |
-| Entity name               | The human-readable name of the entity linked to the alert.                                  |
+| Entity name               | The name of the entity linked to the alert.                                  |
 | Rule UUID                 | A unique identifier for the rule that generated the alert.                                  |
-| Rule name                 | The name assigned to the rule responsible for the alert.                                     |
+| Rule name                 | The name assigned to the rule that triggered the alert.                                     |
 | Status                    | The current state of the alert (e.g., open, acknowledged, resolved).                      |
-| Urgency                   | The level of urgency assigned to the alert, indicating its critical nature.                 |
-| Similar                   | Properties related to similar alerts, facilitating comparative analysis.                     |
+| Urgency                   | The level of urgency assigned to the alert.                 |
+| Similar                   | Properties related to similar alerts.                     |
 | Short ID                  | A concise identifier for quick reference to the alert.                                      |
 | Time to detect            | Duration taken to identify the alert from its occurrence.                                    |
```

Comment:
It would be nice to specify the unit of value of these time* fields. In seconds, if I'm not wrong